sherpa/astro/ui/utils.py

```diff
@@ -2410,10 +2418,10 @@ def set_syserror(self, id, val=None, fractional=False, bkg_id=None):
         val : array or scalar
            The systematic error.
         fractional : bool, optional
-           If ``False`` (the default value), then the ``val`` parameter is
```

Comment:
For reference, the set_staterror call - when rendered in HTML - looks like http://hea-www.cfa.harvard.edu/~dburke/playground/sherpa/ui/api/sherpa.astro.ui.set_staterror.html

(I can't guarantee that this link will remain valid long term, but hopefully this PR will be merged before the link is changed).